skrub/_column_associations.py

```diff
@@ -173,13 +222,14 @@ def _contingency_table(encoded):
 
 
 def _compute_cramer(table, n_samples):
-    """Compute the Cramer V statistic given a contingency table.
+    """Compute the Cramer's V statistic given a contingency table.
 
     The input is the table computed by ``_contingency_table`` with shape
     (n cols, n cols, n bins, n bins).
 
     This returns the symmetric matrix with shape (n cols, n cols) where entry
     i, j contains the statistic for column i x column j.
+
```

Comment:
```suggestion
```

Reply:
for some reason numpy doc does not like an empty line at the end of a docstring -- an annoying quirk